### PR TITLE
feat: Add economy commands

### PR DESCRIPTION
**Pull request by PuneetGopinath**
## Checks

I have...

- [x] Read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [x] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Add economy category!

<del>TODO:</del>Done:
- [x] Add profile cmd
- [x] Add setbio cmd for user to set his w-bot acc's bio. Can be displayed in profile cmd, and in dashboard (dashboard will be added after we get a domain!)
- [x] Add rob cmd
- [x] Add beg cmd (remember: there should be only 1/4 chance for users to get money 💵)
- [x] Change emoji for economy category

Closes #

[contributing]: https://github.com/Welcome-Bot/welcome-bot/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Do you want me to add other economy commands as well in this pr itself?

File: src/commands/economy/daily.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                },
+                disabled: false,
+                cooldown: 10,
+                category: "Economy",
+            },
+            client
+        );
+    }
+
+    async execute({ message, args, guildDB, userDB }, t) {
```
Comment (abhijoshi2k) [inline src/commands/economy/daily.js:30]: `args` is unused argument

File: src/commands/economy/daily.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        const dailyCoins = 100;
+        const time = new Date().getTime();
+        const nextDaily = new Date().setTime(
+            parseInt(userDB.dailyClaimed) - 24 * 60 * 60 * 1000
+        );
+        moment.locale(guildDB.lang ? guildDB.lang.toLowerCase() : "en-US");
+        const momentTime = moment(nextDaily);
```
Comment (abhijoshi2k) [inline src/commands/economy/daily.js:37]: `momentTime` is unused constant

File: src/commands/economy/daily.js (hunk 1 above)
Comment (PuneetGopinath) [reply src/commands/economy/daily.js:30]: We don't need it

**Comment by PuneetGopinath** [comment]
I will try to switch to moment again

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Should we add rob cmd too? Can you do it? It's so simple

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Testing time then review

**Pull request merged**